snuba/datasets/storage.py

```diff
@@ -106,6 +106,21 @@ def get_table_writer(self) -> TableWriter:
         return self.__table_writer
 
 
+class CdcStorage(WritableTableStorage):
```

Comment:
I owuld keep the CdcStorage out of here and in the datasets/cdc module. Everything cdc is there regarding the dataset model. I would argue the cdc concept is not part of the Snuba general dataset api.

Reply:
Moved it into the CDC module